app/src/components/layout/navLinks/NavLinks.tsx

```diff
@@ -8,25 +8,74 @@ import { useSessionContext } from '../../../providers/sessionProvider/SessionPro
 const NavLinks = () => {
     const navigate = useNavigate();
     const [session] = useSessionContext();
-
+    const [width, setWidth] = useState<number>(window.innerWidth);
     const nav = (e: ReactEvent<HTMLAnchorElement, MouseEvent>, link: string) => {
         e.preventDefault();
         navigate(link);
     };
 
+    console.log(width);
+    useEffect(() => {
+        const handleResizeWindow = () => setWidth(window.innerWidth);
+        window.addEventListener('resize', handleResizeWindow);
+        return () => {
+            window.removeEventListener('resize', handleResizeWindow);
+        };
+    }, []);
+
     return session.isLoggedIn ? (
-        <Header.Nav>
-            <Header.NavItem role="link" className="clickable" onClick={(e) => nav(e, routes.HOME)}>
-                Home
-            </Header.NavItem>
+        <Header.Nav style={{ display: 'initial', color: 'white' }}>
             <Header.NavItem
+                style={{ color: 'white' }}
                 role="link"
                 className="clickable"
-                data-testid="logout-btn"
-                onClick={(e) => nav(e, routes.LOGOUT)}
+                onClick={(e) => nav(e, routes.START)}
             >
-                Log Out
+                Home
             </Header.NavItem>
```

Comment:
Suggest to add `href` to each of the NavItems, so that they can be reached by tab and also enter key will fire the onClick event.
(We have a PR that adds keyboard access to buttons and links, but that PR can't cover the change of Nav menu in this branch)